### DISCUSSION TITLE
Fast muter/bypasser - allow selecting groups to be shown

### DISCUSSION THREAD

**Discussion by ssuukk** [rgthree/rgthree-comfy#210] [Ideas]
Some of my groups are purely esthetic, I don't need them to be muted or bypassed, yet they clutter the interface. Could you think about something that would allow selecting which group is part of the UI?

**Comment by rgthree** [comment]
You can! In the Fast Groups Muter/Bypasser nodes’ properties you can filter it to specific colors and/or a Regex match on the group title.
